pkg/kv/bulk/sst_batcher.go

```diff
@@ -725,18 +755,16 @@ func (b *SSTBatcher) addSSTable(
 				br, pErr := b.db.NonTransactionalSender().Send(ctx, ba)
 				sendTime := timeutil.Since(beforeSend)
 
-				atomic.AddInt64(&b.stats.sendWaitAtomic, int64(sendTime))
+				ingestionPerformanceStats.SendWait += sendTime
```

Comment:
I think because this is done on the copy of `currentStats` as opposed to a shared stats variable before, and there's only 1 goroutine modifying this copy, that anything in the `ingestionPerformanceStats` is okay to modify.

Reply:
So `fn` can be called concurrently several times IIUC. We're passing in a pointer to the copy of currentStats and each of the concurrent invocations will mutate that same pointer. Is that not susceptible to concurrent writes?

Reply:
I think I resolved my own question above, if you can just confirm my understanding. Thanks!

Reply:
correct, this `ingestionPerformanceStats` pointer is the pointer to the copy of currentStats, and is only modified by 1 invocation of `fn`